pkg/skaffold/docker/client.go

```diff
@@ -175,6 +199,27 @@ func newMinikubeAPIClient(ctx context.Context, minikubeProfile string) ([]string
 	return environment, api, err
 }
 
+func getMinikubeDriver(ctx context.Context, minikubeProfile string) (string, error) {
+	if minikubeProfile == "" {
+		return "", fmt.Errorf("empty minikube profile")
+	}
```

Comment:
Minikube usually has a default profile "minikube".  I am not sure if Skaffold reads the default profile as "minikube" or just empty string from kubecontext. We will have the check the workflow and make sure it doesn't error out.